<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CHARMMParameterFinderDialog</class>
 <widget class="QDialog" name="CHARMMParameterFinderDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>645</width>
    <height>554</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>CHARMM parameter finder</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>10</y>
     <width>621</width>
     <height>521</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QLabel" name="label_4">
        <property name="font">
         <font>
          <family>Times New Roman</family>
          <pointsize>16</pointsize>
         </font>
        </property>
        <property name="text">
         <string>CHARMM Parameter Finder</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>Times New Roman</family>
         </font>
        </property>
        <property name="text">
         <string>Select collections</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QListWidget" name="collectionsListWidget">
        <property name="selectionMode">
         <enum>QAbstractItemView::MultiSelection</enum>
        </property>
        <property name="currentRow">
         <number>-1</number>
        </property>
        <item>
         <property name="text">
          <string>All-Hydrogen Lipid</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>All-hydrogen ether</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>All-hydrogen carbohydrate</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>All-Hydrogen Proteins</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Silicates</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>General Force Field</string>
         </property>
        </item>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <spacer name="horizontalSpacer_4">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QCheckBox" name="hydrogensCheckBox">
        <property name="text">
         <string>include hydrogens</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_5">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_12">
      <item>
       <widget class="QLabel" name="label_9">
        <property name="font">
         <font>
          <family>Times New Roman</family>
         </font>
        </property>
        <property name="text">
         <string>Time limit:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSlider" name="timeLimitSlider">
        <property name="cursor">
         <cursorShape>SizeHorCursor</cursorShape>
        </property>
        <property name="minimum">
         <number>5</number>
        </property>
        <property name="maximum">
         <number>3000</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="tickPosition">
         <enum>QSlider::TicksBelow</enum>
        </property>
        <property name="tickInterval">
         <number>10</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QwtAnalogClock" name="timeLeftClock" native="true">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>125</height>
           </size>
          </property>
          <property name="lineWidth" stdset="0">
           <number>4</number>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_10">
          <item>
           <spacer name="horizontalSpacer_16">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_7">
            <property name="font">
             <font>
              <family>Times New Roman</family>
             </font>
            </property>
            <property name="text">
             <string>time left</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_17">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QPlainTextEdit" name="reportTextEdit">
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>125</height>
           </size>
          </property>
          <property name="cursor" stdset="0">
           <cursorShape>IBeamCursor</cursorShape>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_11">
          <item>
           <spacer name="horizontalSpacer_18">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QLabel" name="label_8">
            <property name="font">
             <font>
              <family>Times New Roman</family>
             </font>
            </property>
            <property name="text">
             <string>matches</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="horizontalSpacer_19">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <spacer name="horizontalSpacer_3">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="startButton">
        <property name="text">
         <string>Start</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_8">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_8">
      <item>
       <spacer name="horizontalSpacer_9">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="acceptButton">
        <property name="text">
         <string>Accept</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_10">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QPushButton" name="rejectButton">
        <property name="text">
         <string>Reject</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_15">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QwtDial</class>
   <extends>QWidget</extends>
   <header>qwt_dial.h</header>
  </customwidget>
  <customwidget>
   <class>QwtAnalogClock</class>
   <extends>QwtDial</extends>
   <header>qwt_analog_clock.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
